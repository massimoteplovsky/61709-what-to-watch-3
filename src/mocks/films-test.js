export const films = [
  {
    id: 1,
    title: `Terminator 2. Judgment day`,
    genre: [`Action`],
    year: 1991,
    promoPoster: ``,
    poster: ``,
    cover: ``,
    description: ``,
    rating: 8.5,
    votes: 12000,
    director: ``,
    starring: []
  },
  {
    id: 2,
    title: `1+1`,
    genre: [`Drama`, `Comedy`],
    year: 2012,
    promoPoster: ``,
    poster: ``,
    cover: ``,
    description: ``,
    rating: 8.5,
    votes: 12000,
    director: ``,
    starring: []
  },
  {
    id: 3,
    title: `Joker`,
    genre: [`Drama`],
    year: 2019,
    promoPoster: ``,
    poster: ``,
    cover: ``,
    description: ``,
    rating: 8.5,
    votes: 12000,
    director: ``,
    starring: []
  },
  {
    id: 4,
    title: `Once upon a time in Hollywood`,
    genre: [`Drama`],
    year: 2019,
    promoPoster: ``,
    poster: ``,
    cover: ``,
    description: ``,
    rating: 8.5,
    votes: 12000,
    director: ``,
    starring: []
  },
  {
    id: 5,
    title: `Ford vs Ferrari`,
    genre: [`Drama`, `History`],
    year: 2019,
    promoPoster: ``,
    poster: ``,
    cover: ``,
    description: ``,
    rating: 8.5,
    votes: 12000,
    director: ``,
    starring: []
  },
  {
    id: 6,
    title: `1917`,
    genre: [`Drama`, `History`],
    year: 2019,
    promoPoster: ``,
    poster: ``,
    cover: ``,
    description: ``,
    rating: 8.5,
    votes: 12000,
    director: ``,
    starring: []
  },
  {
    id: 7,
    title: `Parasite`,
    genre: [`Drama`, `Comedy`],
    year: 2019,
    promoPoster: ``,
    poster: ``,
    cover: ``,
    description: ``,
    rating: 8.5,
    votes: 12000,
    director: ``,
    starring: []
  },
  {
    id: 8,
    title: `Irishman`,
    genre: [`drama`, `criminal`],
    year: 2019,
    promoPoster: ``,
    poster: ``,
    cover: ``,
    description: ``,
    rating: 8.5,
    votes: 12000,
    director: ``,
    starring: []
  }
];

export const film = {
  id: 1,
  title: `Terminator 2. Judgment day`,
  genre: [`Action`],
  year: 1991,
  promoPoster: ``,
  poster: ``,
  cover: ``,
  description: ``,
  rating: 8.5,
  votes: 12000,
  director: ``,
  starring: []
};

export const promoFilmInfo = {
  title: `The Grand Budapest Hotel`,
  genre: `Drama`,
  year: 2014
};
