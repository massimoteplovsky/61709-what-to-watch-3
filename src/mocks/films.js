export default [
  {
    id: 1,
    title: `Terminator 2. Judgment day`,
    genre: `Action`,
    year: 1991,
    promoPoster: `https://image.tmdb.org/t/p/original/ztBeLd2UCbUATJ1cXwwHev3G3xX.jpg`,
    poster: `http://i4.ytimg.com/vi/HgV7-MJwUBw/mqdefault.jpg`,
    cover: `https://upload.wikimedia.org/wikipedia/en/8/85/Terminator2poster.jpg`,
    description: `In 1995, John Connor is living in Los Angeles with foster parents.
      His mother, Sarah Connor, had been preparing him throughout his childhood for his future role as the
      human resistance leader against Skynet, the artificial intelligence that will be given control of the United States'
      nuclear missiles and initiate a nuclear holocaust on August 29, 1997, known thereafter as "Judgment Day". However,
      Sarah was arrested and imprisoned at a mental hospital after attempting to bomb a computer factory.`,
    rating: 8.5,
    votes: 12000,
    runtime: 120,
    director: `James Cameron`,
    starring: [`Arnold Schwarzenegger`, `Linda Hamilton`, `Robert Patrick`],
    src: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
    reviews: [
      {
        text: `Discerning travellers and Wes Anderson fans will luxuriate in the
         glorious Mittel-European kitsch of one of the director funniest and most exquisitely designed movies in years.`,
        author: `Kate Muir`,
        date: `11.24.2016`,
        rating: 8.6
      },
      {
        text: `Anderson films are too precious for some, but for those of us willing to
        lose ourselves in them, theyre a delight. The Grand Budapest Hotel is no different,
        except that he has added a hint of gravitas to the mix, improving the recipe.`,
        author: `Bill Goodykoontz`,
        date: `10.18.2015`,
        rating: 8.9
      },
      {
        text: `I didn find it amusing, and while I can appreciate the creativity,
        it an hour and 40 minutes I wish I could take back.`,
        author: `Amanda Greever`,
        date: `09.13.2015`,
        rating: 8.0
      },
      {
        text: `Discerning travellers and Wes Anderson fans will luxuriate in the
         glorious Mittel-European kitsch of one of the director funniest and most exquisitely designed movies in years.`,
        author: `Kate Muir`,
        date: `11.24.2016`,
        rating: 8.6
      },
      {
        text: `Anderson films are too precious for some, but for those of us willing to
        lose ourselves in them, theyre a delight. The Grand Budapest Hotel is no different,
        except that he has added a hint of gravitas to the mix, improving the recipe.`,
        author: `Bill Goodykoontz`,
        date: `10.18.2015`,
        rating: 8.9
      },
      {
        text: `I didn find it amusing, and while I can appreciate the creativity,
        it an hour and 40 minutes I wish I could take back.`,
        author: `Amanda Greever`,
        date: `09.13.2015`,
        rating: 8.0
      },
    ]
  },
  {
    id: 2,
    title: `The Intouchables`,
    genre: `Drama`,
    year: 2012,
    promoPoster: `https://i.artfile.ru/1920x1080_1486120_[www.ArtFile.ru].jpg`,
    poster: `https://i.pinimg.com/736x/76/44/c0/7644c07844dd4cb28d2c289bfa6e0dec--the-intouchables-comedy-movies.jpg`,
    cover: `https://upload.wikimedia.org/wikipedia/ru/b/b9/Intouchables.jpg`,
    description: `At night in Paris, Driss (Sy) is driving Philippe's (Cluzet) Maserati
      Quattroporte at high speed. They are chased through the streets by the police and eventually
      cornered. Driss claims the quadriplegic Philippe must be urgently driven to the emergency room;
      Philippe pretends to have a seizure and the fooled police officers escort them to the hospital.`,
    rating: 8.2,
    votes: 1000,
    runtime: 127,
    director: `Olivier Nakache`,
    starring: [`Fran√ßois Cluzet`, `Omar Sy`],
    src: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
    reviews: [
      {
        text: `Discerning travellers and Wes Anderson fans will luxuriate in the
         glorious Mittel-European kitsch of one of the director funniest and most exquisitely designed movies in years.`,
        author: `Kate Muir`,
        date: `11.24.2016`,
        rating: 8.6
      },
      {
        text: `Anderson films are too precious for some, but for those of us willing to
        lose ourselves in them, theyre a delight. The Grand Budapest Hotel is no different,
        except that he has added a hint of gravitas to the mix, improving the recipe.`,
        author: `Bill Goodykoontz`,
        date: `10.18.2015`,
        rating: 8.9
      },
      {
        text: `I didn find it amusing, and while I can appreciate the creativity,
        it an hour and 40 minutes I wish I could take back.`,
        author: `Amanda Greever`,
        date: `09.13.2015`,
        rating: 8.0
      },
      {
        text: `Discerning travellers and Wes Anderson fans will luxuriate in the
         glorious Mittel-European kitsch of one of the director funniest and most exquisitely designed movies in years.`,
        author: `Kate Muir`,
        date: `11.24.2016`,
        rating: 8.6
      },
      {
        text: `Anderson films are too precious for some, but for those of us willing to
        lose ourselves in them, theyre a delight. The Grand Budapest Hotel is no different,
        except that he has added a hint of gravitas to the mix, improving the recipe.`,
        author: `Bill Goodykoontz`,
        date: `10.18.2015`,
        rating: 8.9
      },
      {
        text: `I didn find it amusing, and while I can appreciate the creativity,
        it an hour and 40 minutes I wish I could take back.`,
        author: `Amanda Greever`,
        date: `09.13.2015`,
        rating: 8.0
      },
    ]
  },
  {
    id: 3,
    title: `Joker`,
    genre: `Drama`,
    year: 2019,
    promoPoster: `https://avatars.mds.yandex.net/get-kinopoisk-blog-post-thumb/23341/0511e906546ca5342039bac81f6911f5/orig`,
    poster: `https://all.culture.ru/uploads/9b743d135092e110a8e55a3dc3ad5bc4_w375_h250.jpeg`,
    cover: `https://upload.wikimedia.org/wikipedia/ru/6/6f/%D0%94%D0%B6%D0%BE%D0%BA%D0%B5%D1%80_%28%D1%84%D0%B8%D0%BB%D1%8C%D0%BC_%D0%A2%D0%BE%D0%B4%D0%B4%D0%B0_%D0%A4%D0%B8%D0%BB%D0%BB%D0%B8%D0%BF%D1%81%D0%B0%29.jpg`,
    description: `In the year of 1981 in Gotham City, party
      clown and aspiring stand-up comedian Arthur Fleck lives with his mother, Penny.
      Gotham is rife with crime and unemployment, leaving segments of the population
      disenfranchised and impoverished. Arthur suffers from a medical disorder that causes him
      to laugh at inappropriate times and depends on social services for medication. After a gang
      of delinquents attacks Arthur in an alley, his co-worker, Randall, gives him a gun for self-protection. Arthur acquaints himself with his neighbor,
      single mother Sophie Dumond, and invites her to his upcoming stand-up routine at a nightclub.`,
    rating: 8.1,
    votes: 1440,
    runtime: 118,
    director: `Todd Phillips`,
    starring: [`Joaquin Phoenix`, `Robert De Niro`, `Zazie Beetz`, `Frances Conroy`],
    src: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
    reviews: [
      {
        text: `Discerning travellers and Wes Anderson fans will luxuriate in the
         glorious Mittel-European kitsch of one of the director funniest and most exquisitely designed movies in years.`,
        author: `Kate Muir`,
        date: `11.24.2016`,
        rating: 8.6
      },
      {
        text: `Anderson films are too precious for some, but for those of us willing to
        lose ourselves in them, theyre a delight. The Grand Budapest Hotel is no different,
        except that he has added a hint of gravitas to the mix, improving the recipe.`,
        author: `Bill Goodykoontz`,
        date: `10.18.2015`,
        rating: 8.9
      },
      {
        text: `I didn find it amusing, and while I can appreciate the creativity,
        it an hour and 40 minutes I wish I could take back.`,
        author: `Amanda Greever`,
        date: `09.13.2015`,
        rating: 8.0
      },
      {
        text: `Discerning travellers and Wes Anderson fans will luxuriate in the
         glorious Mittel-European kitsch of one of the director funniest and most exquisitely designed movies in years.`,
        author: `Kate Muir`,
        date: `11.24.2016`,
        rating: 8.6
      },
      {
        text: `Anderson films are too precious for some, but for those of us willing to
        lose ourselves in them, theyre a delight. The Grand Budapest Hotel is no different,
        except that he has added a hint of gravitas to the mix, improving the recipe.`,
        author: `Bill Goodykoontz`,
        date: `10.18.2015`,
        rating: 8.9
      },
      {
        text: `I didn find it amusing, and while I can appreciate the creativity,
        it an hour and 40 minutes I wish I could take back.`,
        author: `Amanda Greever`,
        date: `09.13.2015`,
        rating: 8.0
      },
    ]
  },
  {
    id: 4,
    title: `Once upon a time in Hollywood`,
    genre: `Drama`,
    year: 2019,
    promoPoster: `https://media.kg-portal.ru/production/onceuponatimeinhollywood/onceuponatimeinhollywood_6.jpg`,
    poster: `https://avatars.mds.yandex.net/get-pdb/1945878/3281b62e-4539-4eb7-a26b-fa68f6572bea/s375`,
    cover: `https://upload.wikimedia.org/wikipedia/en/a/a6/Once_Upon_a_Time_in_Hollywood_poster.png`,
    description: `In February 1969, veteran Hollywood actor Rick Dalton (DiCaprio), star of 1950s
      Western television series Bounty Law, fears his career is coming to an end. Casting agent Marvin Schwarz
      (Pacino) advises him to travel to Italy to make Spaghetti Westerns, which Dalton feels are beneath him. `,
    rating: 7.1,
    votes: 440,
    runtime: 122,
    director: `Quentin Tarantino`,
    starring: [`Leonardo DiCaprio`, `Brad Pitt`, `Margot Robbie`],
    src: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
    reviews: [
      {
        text: `Discerning travellers and Wes Anderson fans will luxuriate in the
         glorious Mittel-European kitsch of one of the director funniest and most exquisitely designed movies in years.`,
        author: `Kate Muir`,
        date: `11.24.2016`,
        rating: 8.6
      },
      {
        text: `Anderson films are too precious for some, but for those of us willing to
        lose ourselves in them, theyre a delight. The Grand Budapest Hotel is no different,
        except that he has added a hint of gravitas to the mix, improving the recipe.`,
        author: `Bill Goodykoontz`,
        date: `10.18.2015`,
        rating: 8.9
      },
    ]
  },
  {
    id: 5,
    title: `Ford vs Ferrari`,
    genre: `Drama`,
    year: 2019,
    promoPoster: `https://alenarus.ru/wp-content/uploads/2019/12/ford-v-ferrari.jpg`,
    poster: `https://sun9-28.userapi.com/c206816/v206816832/1771c/3p_wj8sfMz8.jpg`,
    cover: `https://upload.wikimedia.org/wikipedia/en/a/a4/Ford_v._Ferrari_%282019_film_poster%29.png`,
    description: `In 1963, Ford Motor Company Vice President, Lee Iacocca, proposes to Henry Ford II
      to purchase the cash-strapped Ferrari as a means to boost their car sales by participating in the 24 Hours of Le Mans.
      Enzo Ferrari, however, walks out of the deal, as Fiat offers him a more lucrative deal that allows him to retain his ownership of
      Scuderia Ferrari. In rejecting the proposed deal with Ford, Ferrari insults the company and its CEO. `,
    rating: 8.1,
    votes: 940,
    runtime: 157,
    director: `James Mangold`,
    starring: [`Matt Damon`, `Christian Bale`],
    src: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
    reviews: [
      {
        text: `Discerning travellers and Wes Anderson fans will luxuriate in the
         glorious Mittel-European kitsch of one of the director funniest and most exquisitely designed movies in years.`,
        author: `Kate Muir`,
        date: `11.24.2016`,
        rating: 8.6
      },
      {
        text: `Anderson films are too precious for some, but for those of us willing to
        lose ourselves in them, theyre a delight. The Grand Budapest Hotel is no different,
        except that he has added a hint of gravitas to the mix, improving the recipe.`,
        author: `Bill Goodykoontz`,
        date: `10.18.2015`,
        rating: 8.9
      },
      {
        text: `I didn find it amusing, and while I can appreciate the creativity,
        it an hour and 40 minutes I wish I could take back.`,
        author: `Amanda Greever`,
        date: `09.13.2015`,
        rating: 8.0
      },
      {
        text: `Discerning travellers and Wes Anderson fans will luxuriate in the
         glorious Mittel-European kitsch of one of the director funniest and most exquisitely designed movies in years.`,
        author: `Kate Muir`,
        date: `11.24.2016`,
        rating: 8.6
      },
      {
        text: `Anderson films are too precious for some, but for those of us willing to
        lose ourselves in them, theyre a delight. The Grand Budapest Hotel is no different,
        except that he has added a hint of gravitas to the mix, improving the recipe.`,
        author: `Bill Goodykoontz`,
        date: `10.18.2015`,
        rating: 8.9
      },
      {
        text: `I didn find it amusing, and while I can appreciate the creativity,
        it an hour and 40 minutes I wish I could take back.`,
        author: `Amanda Greever`,
        date: `09.13.2015`,
        rating: 8.0
      },
    ]
  },
  {
    id: 6,
    title: `1917`,
    genre: `Drama`,
    year: 2019,
    promoPoster: `https://f.otzyv.ru/f/kino/2019/24272/25111905502941.jpg`,
    poster: `https://sun3-13.userapi.com/c857724/v857724955/14e882/u5CFMvbvX3I.jpg`,
    cover: `https://upload.wikimedia.org/wikipedia/en/7/7a/1917poster.jpg`,
    description: `On 6 April 1917, aerial reconnaissance has observed that
      the German army, which has pulled back from a sector of the Western Front in northern France,
      is not in retreat but has made a strategic withdrawal to the new Hindenburg Line,
      where they are waiting to overwhelm the British with artillery. `,
    rating: 8.1,
    votes: 280,
    runtime: 115,
    director: `Sam Mendes`,
    starring: [`George MacKay`, `Dean-Charles Chapman`, `Mark Strong`],
    src: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
    reviews: [
      {
        text: `Discerning travellers and Wes Anderson fans will luxuriate in the
         glorious Mittel-European kitsch of one of the director funniest and most exquisitely designed movies in years.`,
        author: `Kate Muir`,
        date: `11.24.2016`,
        rating: 8.6
      },
      {
        text: `Anderson films are too precious for some, but for those of us willing to
        lose ourselves in them, theyre a delight. The Grand Budapest Hotel is no different,
        except that he has added a hint of gravitas to the mix, improving the recipe.`,
        author: `Bill Goodykoontz`,
        date: `10.18.2015`,
        rating: 8.9
      },
      {
        text: `I didn find it amusing, and while I can appreciate the creativity,
        it an hour and 40 minutes I wish I could take back.`,
        author: `Amanda Greever`,
        date: `09.13.2015`,
        rating: 8.0
      },
      {
        text: `Discerning travellers and Wes Anderson fans will luxuriate in the
         glorious Mittel-European kitsch of one of the director funniest and most exquisitely designed movies in years.`,
        author: `Kate Muir`,
        date: `11.24.2016`,
        rating: 8.6
      },
      {
        text: `Anderson films are too precious for some, but for those of us willing to
        lose ourselves in them, theyre a delight. The Grand Budapest Hotel is no different,
        except that he has added a hint of gravitas to the mix, improving the recipe.`,
        author: `Bill Goodykoontz`,
        date: `10.18.2015`,
        rating: 8.9
      },
      {
        text: `I didn find it amusing, and while I can appreciate the creativity,
        it an hour and 40 minutes I wish I could take back.`,
        author: `Amanda Greever`,
        date: `09.13.2015`,
        rating: 8.0
      },
    ]
  },
  {
    id: 7,
    title: `Parasite`,
    genre: `Drama`,
    year: 2019,
    promoPoster: `https://avatars.mds.yandex.net/get-afishanew/29882/656a09f76572753b87ae7ddb9434e7c6/orig`,
    poster: `http://m.kino-teatr.ru/art/5395/pv_75950.jpg`,
    cover: `https://upload.wikimedia.org/wikipedia/en/5/53/Parasite_%282019_film%29.png`,
    description: `On 6 April 1917, aerial reconnaissance has observed that the German army,
      which has pulled back from a sector of the Western Front in northern France, is not in retreat
      but has made a strategic withdrawal
      to the new Hindenburg Line, where they are waiting to overwhelm the British with artillery. `,
    rating: 8.3,
    votes: 563,
    runtime: 121,
    director: `Bong Joon-ho`,
    starring: [`Song Kang-ho`, `Lee Sun-kyun`, `Cho Yeo-jeong`],
    src: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
    reviews: [
      {
        text: `Discerning travellers and Wes Anderson fans will luxuriate in the
         glorious Mittel-European kitsch of one of the director funniest and most exquisitely designed movies in years.`,
        author: `Kate Muir`,
        date: `11.24.2016`,
        rating: 8.6
      },
      {
        text: `Anderson films are too precious for some, but for those of us willing to
        lose ourselves in them, theyre a delight. The Grand Budapest Hotel is no different,
        except that he has added a hint of gravitas to the mix, improving the recipe.`,
        author: `Bill Goodykoontz`,
        date: `10.18.2015`,
        rating: 8.9
      },
      {
        text: `I didn find it amusing, and while I can appreciate the creativity,
        it an hour and 40 minutes I wish I could take back.`,
        author: `Amanda Greever`,
        date: `09.13.2015`,
        rating: 8.0
      },
      {
        text: `Discerning travellers and Wes Anderson fans will luxuriate in the
         glorious Mittel-European kitsch of one of the director funniest and most exquisitely designed movies in years.`,
        author: `Kate Muir`,
        date: `11.24.2016`,
        rating: 8.6
      },
      {
        text: `Anderson films are too precious for some, but for those of us willing to
        lose ourselves in them, theyre a delight. The Grand Budapest Hotel is no different,
        except that he has added a hint of gravitas to the mix, improving the recipe.`,
        author: `Bill Goodykoontz`,
        date: `10.18.2015`,
        rating: 8.9
      },
      {
        text: `I didn find it amusing, and while I can appreciate the creativity,
        it an hour and 40 minutes I wish I could take back.`,
        author: `Amanda Greever`,
        date: `09.13.2015`,
        rating: 8.0
      },
    ]
  },
  {
    id: 8,
    title: `Irishman`,
    genre: `Drama`,
    year: 2019,
    promoPoster: `https://static.tildacdn.com/tild6431-3466-4234-b065-313635616562/_7.jpg`,
    poster: `https://sun9-31.userapi.com/c850608/v850608676/40eb4/ojB5App6exk.jpg`,
    cover: `https://upload.wikimedia.org/wikipedia/ru/thumb/a/a2/The_Irishman_%28poster%29.jpg/345px-The_Irishman_%28poster%29.jpg`,
    description: `In 1950s Philadelphia, Sheeran works as a delivery truck driver and starts to sell some of the contents of his shipments to local gangster Felix "Skinny Razor" DiTullio. After his company accuses him of theft, union lawyer Bill Bufalino gets him off after Sheeran refuses to name his customers to the judge.
      Bill introduces Sheeran to his cousin Russell Bufalino,
      head of the Northeastern Pennsylvania crime family. `,
    rating: 7.5,
    votes: 380,
    runtime: 200,
    director: `Martin Scorsese`,
    starring: [`Robert De Niro`, `Al Pacino`, `Joe Pesci`],
    src: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
    reviews: [
      {
        text: `Discerning travellers and Wes Anderson fans will luxuriate in the
         glorious Mittel-European kitsch of one of the director funniest and most exquisitely designed movies in years.`,
        author: `Kate Muir`,
        date: `11.24.2016`,
        rating: 8.6
      },
      {
        text: `Anderson films are too precious for some, but for those of us willing to
        lose ourselves in them, theyre a delight. The Grand Budapest Hotel is no different,
        except that he has added a hint of gravitas to the mix, improving the recipe.`,
        author: `Bill Goodykoontz`,
        date: `10.18.2015`,
        rating: 8.9
      },
      {
        text: `I didn find it amusing, and while I can appreciate the creativity,
        it an hour and 40 minutes I wish I could take back.`,
        author: `Amanda Greever`,
        date: `09.13.2015`,
        rating: 8.0
      },
      {
        text: `Discerning travellers and Wes Anderson fans will luxuriate in the
         glorious Mittel-European kitsch of one of the director funniest and most exquisitely designed movies in years.`,
        author: `Kate Muir`,
        date: `11.24.2016`,
        rating: 8.6
      },
      {
        text: `Anderson films are too precious for some, but for those of us willing to
        lose ourselves in them, theyre a delight. The Grand Budapest Hotel is no different,
        except that he has added a hint of gravitas to the mix, improving the recipe.`,
        author: `Bill Goodykoontz`,
        date: `10.18.2015`,
        rating: 8.9
      },
      {
        text: `I didn find it amusing, and while I can appreciate the creativity,
        it an hour and 40 minutes I wish I could take back.`,
        author: `Amanda Greever`,
        date: `09.13.2015`,
        rating: 8.0
      },
    ]
  }
];
