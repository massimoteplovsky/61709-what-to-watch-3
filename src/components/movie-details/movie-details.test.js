import React from "react";
import renderer from "react-test-renderer";
import {film} from "../../mocks/films-test";
import MovieDetails from "./movie-details.jsx";

it(`<MovieDetails /> component renders correctly`, () => {
  const tree = renderer
    .create(<MovieDetails
      filmInfo={film}
    />);

  expect(tree).toMatchSnapshot();
});
