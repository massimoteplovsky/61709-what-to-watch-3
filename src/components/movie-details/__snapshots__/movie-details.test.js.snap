// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieDetails /> component renders correctly 1`] = `
<div
  className="movie-card__text movie-card__row"
>
  <div
    className="movie-card__text-col"
  >
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Director
      </strong>
      <span
        className="movie-card__details-value"
      >
        
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Starring
      </strong>
      <span
        className="movie-card__details-value"
      >
        Tim cook

      </span>
    </p>
  </div>
  <div
    className="movie-card__text-col"
  >
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Run Time
      </strong>
      <span
        className="movie-card__details-value"
      >
        0h 4m
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Genre
      </strong>
      <span
        className="movie-card__details-value"
      >
        Action
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Released
      </strong>
      <span
        className="movie-card__details-value"
      >
        1991
      </span>
    </p>
  </div>
</div>
`;
